docs/install/index.md

```diff
@@ -72,9 +72,13 @@ dotnet add package Microsoft.AI.MachineLearning
 
 ## Install on web and mobile
 
-The installation instructions in this section use pre-built packages that include support for selected operators and ONNX opset versions based on the requirements of popular models. Your model must only use the [opsets and operators](../reference/operators/mobile_package_op_type_support_1.9.md) supported by the pre-built package.
+Unless stated otherwise, the installation instructions in this section refer to pre-built packages that include support for selected operators and ONNX opset versions based on the requirements of popular models. These packages may be referred to as "mobile packages". If you use mobile packages, your model must only use the supported [opsets and operators](../reference/operators/mobile_package_op_type_support_1.9.md).
 
-If the pre-built package is too large, or does not include the operators in your model/s, you can create a [custom build](../build/custom.md).
+Another type of pre-built package has full support for all ONNX opsets and operators, at the cost of larger binary size. These packages are referred to as "full packages".
+
+If the pre-built mobile package is too large, you can create a [custom build](../build/custom.md).
+
+If the pre-built mobile package does not include the operators in your model/s, you can either use the full package if available, or create a custom build.
 
```

Comment:
Could also be the model opset is the issue. Can add link when the new doco on the onnx opset update helper is checked in.